### DISCUSSION TITLE
How can I use ProLIF to analyse protein-ligand interactions in Gromacs MD?

### DISCUSSION THREAD

**Discussion by nachotu** [chemosim-lab/ProLIF#72] [Q&A]
Hello everyone, I'd like to use ProLIF to make a protein-ligand fingerprint from a gromacs MD simulation. Someone could help me? 
Thank you all!

**Comment by cbouy** [comment]
Hi @nachotu ,

Have you had a look at the [quickstart tutorial](https://prolif.readthedocs.io/en/latest/notebooks/quickstart.html)? 
In the first cell at the beginning:  
```python
import MDAnalysis as mda
import prolif as plf
# load trajectory
u = mda.Universe(plf.datafiles.TOP, plf.datafiles.TRAJ)
# create selections for the ligand and protein
lig = u.atoms.select_atoms("resname LIG")
prot = u.atoms.select_atoms("protein")
```

just replace `plf.datafiles.TOP` with a path to your Gromacs topology, and `plf.datafiles.TRAJ` with a path to your trajectory.

[MDAnalysis](https://docs.mdanalysis.org/stable/documentation_pages/overview.html) (which is used for reading the inputs) supports a wide range of formats including those from Gromacs so it should work out of the box.
